sql-plugin/src/main/scala-2.12/com/nvidia/spark/rapids/RapidsStack.scala

```diff
@@ -18,10 +18,8 @@ package com.nvidia.spark.rapids
 
 import scala.collection.mutable.ArrayStack
 
-class RapidsStack[T] extends Proxy {
-  private val stack = new ArrayStack[T]()
-
-  override def self = stack
+class RapidsStack[T] {
+  val self: ArrayStack[T] = new ArrayStack[T]()
```

Comment:
Nit: No need for this to be public.  Comment applies to all the other instances as well.